packages/entity-detail/src/modules/entityDetail/sagas.js

```diff
@@ -76,16 +76,17 @@ export function* loadEntityModel(entityName) {
 }
 
 export function* loadDetailView() {
-  const {entityName, formName, mode} = yield select(entityDetailSelector)
+  const {entityName, formName, mode, defaultValues} = yield select(entityDetailSelector)
```

Comment:
We could name these `storeDefaultValues` and `definitionDefaultValues` to avoid numbers in naming?